deploy/cdk/lib/foundation/foundation-stack.ts

```diff
@@ -114,6 +114,11 @@ export class FoundationStack extends Stack {
       lifecycleRules: [{ enabled: true, expiration: Duration.days(365 * 10), noncurrentVersionExpiration: Duration.days(1) }],
     })
 
+    new CfnOutput(this, 'ExportsOutputFnGetAttLogBucketCC3B17E8DomainName1D649D3C', {
```

Comment:
Just want to note that we're having to explicitly add this export because aws-cdk reverted a previous decision about what bucket property Cloudfront will use when using a log bucket: https://github.com/aws/aws-cdk/pull/10945. It's being reverted to use `bucket.bucketRegionalDomainName`, so CloudFormation will try to delete the previous export cdk created that was associated with `bucket. bucketDomainName` and will fail because dependent stacks are using them.